addons/mail/models/mail_thread.py

```diff
@@ -2223,7 +2223,7 @@ def _notify_thread_by_email(self, message, recipients_data, msg_vals=False,
                                 mail_auto_delete=True, # mail.mail
                                 model_description=False, force_email_company=False, force_email_lang=False,  # rendering
                                 check_existing=False, force_send=True, send_after_commit=True,  # email send
-                                **kwargs):
+                                force_subtitles=None, **kwargs):
```

Comment:
why not just `subtitles` ?

Reply:
Because it overrides "subtitles" value potentially returned by _notify_by_email_prepare_rendering_context. I have added a comment in the documentation of that parameter to make it clear: optional list that will be set as template value "subtitles" (overrides subtitles returned by _notify_by_email_prepare_rendering_context if any)
Do you think it is justified or you prefer subtitles ?

Reply:
I slightly prefer subtitles, but not a big deal anyway :)

Reply:
I have changed it, it makes the lines shorter